tests/benchmarks/test_subscriptions.py

```diff
@@ -24,3 +26,25 @@ async def _run():
             assert value.data["something"] == "Hello World!"
 
     benchmark(lambda: asyncio.run(_run()))
```

Comment:
do you think we still need this test now? 🤔

Reply:
maybe. This one tests the whole lifespan of a subscription while the other is for the subscription itself.

Reply:
let's do that!

I fixed the type issue, it's on main :D